aspect/intellij_info_impl.bzl

```diff
@@ -4,6 +4,9 @@ load(
     "@bazel_tools//tools/build_defs/cc:action_names.bzl",
     "ACTION_NAMES",
 )
+load("@rules_cc//cc/common:cc_common.bzl", "cc_common")
+load("@rules_cc//cc/common:cc_info.bzl", "CcInfo")
+load("@rules_java//java/common:java_common.bzl", "java_common")
```

Comment:
This will not work in the aspects. If we really need these load statements here we need to template this for comparability reasons.

Reply:
Ack. I reverted this part of the change. So the remaining change is still generated by buildifier, but it shouldn't break any functionality. Let's take this part to a second smaller PR.

Reply:
I agree, let's upgrade the aspects separately since they might require more work.